package.json

```
@@ -11,7 +11,7 @@
     "@sentry/tracing": "^7.0.0",
     "@sentry/webpack-plugin": "^1.18.3",
     "@types/dompurify": "^2.0.3",
-    "@types/node": "^12",
+    "@types/node": "^16",
```

```suggestion
    "@types/node": "^14",
```

This should fix CI hopefully